extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -278,7 +274,7 @@ define('ext.wikia.adEngine.template.porvata', [
 
 			onReady(video, params);
 
-			if (prebid && params.useBidAsFallback) {
+			if (bidders && params.useBidAsFallback) {
```

Comment:
Should we check whether `bidders` are enabled?

Reply:
Yes we should, fixed.